src/ami-set.js

```diff
@@ -0,0 +1,23 @@
+let base = require('taskcluster-base');
+
+const KEY_CONST = 'ami-set';
+
+ /**
+ * An AMI Set is a collection of AMIs with a single name (its AMI Set ID).
+ * Each AMI in the set is keyed by its virtualization type (PV or HVM) and
+ * by its AWS region.
+ */
+
+let amiSet = base.Entity.configure({
```

Comment:
This should be capitalized (`AmiSet`).  The idea is that this is a class, so just like in Python we use an initial capital letter.